system/Config/Services.php

```diff
@@ -96,7 +97,7 @@ public static function cache(\Config\Cache $config = null, $getShared = true)
 			return self::getSharedInstance('cache', $config);
 		}
 
-		if (! is_object($config))
+		if (!is_object($config))
```

Comment:
NetBeans provides for spaces around unary operators, leading to `( ! is_object...`.
The same setting results in `$i ++`.
Is this ok, or do you have a workaround (other than switching to phpstorm)?